src/storage/diffusion.md

```diff
@@ -0,0 +1,26 @@
+# Chain diffusion
+
+The Ouroboros Consensus Protocols assume instantaneous transmission (and
+validation) of chains, becoming instantly available even for newly joined
+peers. This is, even if just because of physical real-world limitations,
+infeasible in practice. For this reason, transmission of chains is done in a
+block-by-block basis instead. This allows chains to be incrementally sent to
+peers but comes with its own risks for newly joined peers (see [Ouroboros
+Genesis](./chainsel.md#ouroboros-genesis)).
+
+Furthermore, blocks (which in the end is the data transmitted over the network)
+are subdivided in block headers and block bodies, as described in the
+[Header|body split](./#headerbody-split) section.
+
+The diffusion of data in the Cardano network is in fact a responsibility of the
+[Networking layer](../network). The diffusion of chains involves accessing the
+Storage layer and serving its contents to peers, however it is ultimately the
+[Consensus layer](../consensus/) the one that decides how the data in the
+Storage layer is mutated, as an outcome of [Chain Selection](../chainsel.md).
+
+> Chain diffusion is a joint effort of the Consensus, Network and Storage
```

Comment:
I like these highlights. Is there a suitable marker we could use (tip, attention, etc).

Reply:
This one in particular felt more important than a tip or a warning. That's why I used a simple quote which is very highlighted imho

Reply:
You can give it even more emphasis by using alerts like so:

> [!important]
> something important

See https://cardano-scaling.github.io/cardano-blueprint/styleguide.html#alerts